proxy/src/transparency/glue.rs

```diff
@@ -124,57 +144,41 @@ impl<B> BodyStream<B> {
     pub fn new(body: B) -> Self {
         BodyStream {
             body,
-            poll_trailers: false,
         }
     }
 }
 
-impl<B> Stream for BodyStream<B>
+// NOTE: I think it's possible the `BodyStream` type can be removed in favour
```

Comment:
Can we make a call on this before merging and remove this NOTE?

Reply:
I think the comment may not be true, I'll just remove it.